sops/PKGBUILD

```
@@ -19,6 +19,8 @@ prepare() {
 
 build() {
   cd src/go.mozilla.org/sops
+  go mod download
+  make vendor
   env GOPATH="${srcdir}" go build \
```

```suggestion
   env GO111MODULE=on GOPATH="${srcdir}" go build \
```
Can you try that? (without your other changes please)
Sops switched to using modules for this release

---

It's work :)